examples/examples_nodejs_test.go

```diff
@@ -322,7 +332,7 @@ func TestAccServerlessRaw(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir:           filepath.Join(getCwd(t), "serverless-raw"),
-			RunUpdateTest: true,
+			RunUpdateTest: false,
```

Comment:
I updated the test to use the new enums for `ManagedPolicy`. According to @stack72 if this is set to `true` it tests against the published pulumi-aws first and since these enums aren't released yet the test would fail. Can set this back once we get these out.